Source/WebCore/Modules/model-element/HTMLModelElement.cpp

```diff
@@ -856,7 +856,7 @@ void HTMLModelElement::environmentMapRequestResource()
     options.destination = FetchOptions::Destination::Environmentmap;
 
     auto crossOriginAttribute = parseCORSSettingsAttribute(attributeWithoutSynchronization(HTMLNames::crossoriginAttr));
-    auto request = createPotentialAccessControlRequest(ResourceRequest { m_environmentMapURL }, WTFMove(options), document(), crossOriginAttribute);
+    auto request = createPotentialAccessControlRequest(ResourceRequest { URL { m_environmentMapURL } }, WTFMove(options), document(), crossOriginAttribute);
```

Comment:
You've already landed this, but this is the correct change. Thank you for fixing!